src/stories/standalone/UIKit/Checkbox.story.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { withActions } from "@storybook/addon-actions";
+import { boolean, text, withKnobs } from "@storybook/addon-knobs";
+
+import FormGroup from "@material-ui/core/FormGroup";
+import FormControlLabel from "@material-ui/core/FormControlLabel";
+// import StyledCheckbox from "../../../standalone/UIKit/Checkbox";
+import Checkbox from "../../../standalone/UIKit/Checkbox";
+
+export const CheckboxStory = (): React.ReactElement => {
+	const [state, setState] = React.useState({
```

Comment:
state and handleChange can be simplified, see the official Material-UI example: https://github.com/mui-org/material-ui/blob/master/docs/src/pages/components/checkboxes/Checkboxes.tsx